### PR TITLE
the design is partially made as well as the file structure

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

**Comment by DmitriyKrivovyaz** [comment]
Screenshot_6

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  info: {
+    linkName: 'Info',
+    path: INFO_ROUTE
+  },
+  feedback: {
+    linkName: 'Feedback',
+    path: FEEDBACK_ROUTE
+  }
+};
```
Comment (glebdronchenko) [inline src/components/header/Header.jsx:19]: Variables _links_ and _rotes_ are the same. You can combine them.

File: src/index.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import ReactDOM from 'react-dom';
 import App from './App.jsx';
+import './index.css';
```
Comment (glebdronchenko) [inline src/index.jsx:3]: You need to use the same style guide.

File: src/components/home/home.module.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.homePageContainer {
+    /* background-color: aliceblue; */
+    height: calc(100vh - 70px);
```
Comment (glebdronchenko) [inline src/components/home/home.module.css]: Try not to use _calc()_

File: src/components/generic/backgroundVideo/BackgroundVideo.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import backgroundVideo from '../../../../public/static/videos/background.mp4';
```
Comment (glebdronchenko) [inline src/components/generic/backgroundVideo/BackgroundVideo.jsx:2]: So long path. You need to use shorter path.

File: src/components/home/currentWeather/CurrentWeather.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import classes from './currentWeather.module.css'
+import sunny from '../../../../public/static/photos/weather/sunny.png'
```
Comment (glebdronchenko) [inline src/components/home/currentWeather/CurrentWeather.jsx:3]: You need to use shorter path.

File: src/components/home/Home.jsx [hunk 6]
```diff
@@ -1,12 +1,42 @@
 import React from 'react';
-import classes from './home.module.css'
+import CurrentWeather from './currentWeather/CurrentWeather';
+import DayWeatherBox from './dayWeatherBox/DayWeatherBox';
+import classes from './home.module.css';
+import HourSlider from './hourSlider/HourSlider';
+
+const city = 'Minsk';
+const title = `Weather in ${city}`
```
Comment (glebdronchenko) [inline src/components/home/Home.jsx:8]: The variables have to be inside of component.

File: src/utils/consts.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export const HOME_ROUTE = '/';
+export const FEEDBACK_ROUTE = '/feedback';
+export const INFO_ROUTE = '/info';
```
Comment (glebdronchenko) [inline src/utils/consts.js:3]: This values of constants can be in router variable.

**Review by glebdronchenko — COMMENTED** [review]
My remarks written in comments to commits.

Name of branch is wrong. You need to change it.

File: src/components/home/Home.jsx [hunk 8]
```diff
@@ -1,12 +1,42 @@
...
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+              <DayWeatherBox/>
+            </div>
```
Comment (glebdronchenko) [inline src/components/home/Home.jsx:35]: Maybe you need to add wrapper component for _DayWeatherBox_ component.

File: src/components/header/Header.jsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+function Header() {
+  return (
+    <div className={classes.header}>
+      <div className={classes.headerLeftpart}>
+        <DefaultLink link={links.home}></DefaultLink>
+      </div>
+      <div className={classes.headerRightPart}>
+        <DefaultLink link={links.info}></DefaultLink>
+        <DefaultLink link={links.feedback}></DefaultLink>
+      </div>
```
Comment (glebdronchenko) [inline src/components/header/Header.jsx:30]: Maybe you need to add nav-wrapper component for menu links.

**Review by glebdronchenko — COMMENTED** [review]
You need to make of separating of components more.

**Pull request closed without merging**